modules/osdk-hh-helm-api-logic.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/helm/osdk-hybrid-helm.adoc
+
+:_content-type: CONCEPT
+[id="osdk-hh-helm-api-logic_{context}"]
+= Operator logic for the Helm API
+
+By default, the scaffolded Operator watches `Memcached` resource events as shown in the `watches.yaml` file and executes Helm releases using the specified chart.
```

Comment:
It's probably because I don't know what _scaffold_ means here, but we've talked about scaffolding an API and Helm charts. Nothing about scaffolding an Operator. 
Also, where this is a separate module, should you use the full name of the Operator?

Reply:
Went with "your scaffolded Operator project" here.